irods/test/pool_test.py

```diff
@@ -237,6 +241,76 @@ def test_no_refresh_connection(self):
             self.assertEqual(0, len(self.sess.pool.active))
             self.assertEqual(0, len(self.sess.pool.idle))
 
+    # Test to confirm the connection destructor log message is actually
+    # logged to file, to confirm the destructor is called
+    def test_connection_destructor_called(self):
+        # Set 'irods_connection_refresh_time' to '3' (in seconds) in
+        # ~/.irods/irods_environment.json file. This means any connection
+        # that was created more than 3 seconds ago will be dropped and
+        # a new connection is created/returned. This is to avoid
+        # issue with idle connections that are dropped.
+        conn_obj_id_1 = None
+        conn_obj_id_2 = None
+        create_time_1 = None
+        create_time_2 = None
+        last_used_time_1 = None
+        last_used_time_2 = None
+
+        try:
+
+           # Create a temporary log file
+           my_log_file = tempfile.NamedTemporaryFile()
+
+            logging.getLogger('irods.connection').setLevel(logging.DEBUG)
+            file_handler = logging.FileHandler(my_log_file.name, mode='a')
+            file_handler.setLevel(logging.DEBUG)
+            logging.getLogger('irods.connection').addHandler(file_handler)
```

Comment:
Codacy caught that these lines have an extra two spaces (unexpected indent). Good job, robots.

Reply:
Fixed the indentation. Thanks!